src/query/xasl_to_stream.c

```diff
@@ -4614,13 +4614,6 @@ xts_process_cls_spec_type (char *ptr, const CLS_SPEC_TYPE * cls_spec)
     }
   ptr = or_pack_int (ptr, offset);
 
-  offset = xts_save_regu_variable_list (cls_spec->cls_regu_list_last_version);
-  if (offset == ER_FAILED)
-    {
-      return NULL;
-    }
-  ptr = or_pack_int (ptr, offset);
-
   offset = xts_save_outptr_list (cls_spec->cls_output_val_list);
```

Comment:
Please also fix `xts_sizeof_cls_spec_type` function. 
It includes space for `cls_regu_list_last_version`. 

```
  size += (PTR_SIZE             /* cls_regu_list_key */
           + PTR_SIZE           /* cls_regu_list_pred */
           + PTR_SIZE           /* cls_regu_list_rest */
           + PTR_SIZE           /* cls_regu_list_range */
           + PTR_SIZE           /* cls_regu_list_last_version */
           + PTR_SIZE           /* cls_output_val_list */
           + PTR_SIZE           /* regu_val_list */
           + OR_HFID_SIZE       /* hfid */
           + OR_OID_SIZE        /* cls_oid */
           + OR_INT_SIZE        /* num_attrs_key */
           + PTR_SIZE           /* attrids_key */
           + PTR_SIZE           /* cache_key */
           + OR_INT_SIZE        /* num_attrs_pred */
           + PTR_SIZE           /* attrids_pred */
           + PTR_SIZE           /* cache_pred */
           + OR_INT_SIZE        /* num_attrs_rest */
           + PTR_SIZE           /* attrids_rest */
           + PTR_SIZE           /* cache_rest */
           + OR_INT_SIZE        /* schema_type */
           + OR_INT_SIZE        /* num_attrs_reserved */
           + PTR_SIZE           /* cache_reserved */
           + PTR_SIZE           /* cls_regu_list_reserved */
           + PTR_SIZE           /* atrtrids_range */
           + PTR_SIZE           /* cache_range */
           + OR_INT_SIZE);      /* num_attrs_range */

```